big_o/big_o.py

```diff
@@ -173,3 +173,52 @@ def big_o(func, data_generator,
         fitted['times'] = time
 
     return best, fitted
+
+def print_report(best, others):
```

Comment:
As I had suggested, please call this function `big_o_report`, and move it to a new module, `reports.py`.

The function should build and return a string instead of printing directly. This is so that the report can be logged or printed to any output stream. Or even emailed, if necessary.

Reply:
module created